packages/workbox-cli/test/node/generate-sw-e2e.js

```diff
@@ -17,7 +17,7 @@ describe('Generate SW End-to-End Tests', function() {
   const FILE_EXTENSIONS = ['html', 'css', 'js', 'png'];
 
   before(function() {
-    tmpDirectory = fs.mkdtempSync(path.join(__dirname, 'tmp-'));
+    tmpDirectory = fs.mkdtempSync(path.join(__dirname, 'temp-'));
```

Comment:
Sorry, the tickles were me trying to get Travis CI (and then later AppVeyor) to recognize the PR following the repo renaming. I had to push a harmless commit.